pages/admin/admin-management.tsx

```diff
@@ -1,15 +1,137 @@
+/* eslint-disable @typescript-eslint/no-empty-function */
+// Eslint Disable Temporarily for Pagination since API hookup not complete.
 import { GetServerSideProps } from 'next'; // Get server side props
 import { getSession } from 'next-auth/client'; // Session management
-import { Text, GridItem } from '@chakra-ui/react'; // Chakra UI
 import Layout from '@components/internal/Layout'; // Layout component
 import { authorize } from '@tools/authorization'; // Page authorization
+import Table from '@components/internal/Table'; // Table component
 
+import {
+  Flex,
+  Text,
+  Box,
+  GridItem,
+  Menu,
+  MenuButton,
+  MenuList,
+  MenuItem,
+  IconButton,
+  Button,
+  Select,
+} from '@chakra-ui/react'; // Chakra UI
 // Internal home page
+import { AddIcon } from '@chakra-ui/icons'; // Chakra UI icons
+import Pagination from '@components/internal/Pagination'; // Pagination component
+
+function renderActionsMenu() {
+  return (
+    <Menu>
+      <MenuButton
+        as={IconButton}
+        aria-label="Options"
+        icon={<img src="/assets/three-dots.svg" />}
+        variant="outline"
+        border="none"
+      />
+      <MenuList>
+        <MenuItem>Edit User</MenuItem>
+        <MenuItem color="text.critical" textStyle="button-regular">
+          Delete User
+        </MenuItem>
+      </MenuList>
+    </Menu>
+  );
+}
+
+// Render the roles menu
+function renderRoleMenu({ value }: { value: string }) {
+  return (
+    <Select defaultValue={value} width={190}>
+      <option value="frontDesk">Front Desk</option>
+      <option value="accountant">Accountant</option>
+      <option value="admin">Admin</option>
+    </Select>
+  );
+}
```

Comment:
Can you move these functions to inside the component below, and rename them to `_renderActionsMenu` and `_renderRoleMenu`? Also please add a comment to the `_renderActionsMenu` function like that for the `_renderRoleMenu` function you had above.

Reply:
https://github.com/uwblueprint/richmond-centre-for-disability/pull/63/commits/39605957605083d6793db5953bac64ed105a2693